apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -0,0 +1,688 @@
+import { onDocumentReady } from '@prairielearn/browser-utils';
+import { BootstrapTableOptions } from 'bootstrap-table';
+
+declare global {
+  interface Window {
+    popoverSubmitViaAjax: (e: any, popover: JQuery) => void;
+  }
+}
+
+declare global {
+  interface JQuery {
+    bootstrapTable(options: BootstrapTableOptions): JQuery;
+    bootstrapTable(method: string, ...parameters: any[]): JQuery | any;
+  }
+}
```

Comment:
Rather than defining your own types, you can add `bootstrap-table` here:

https://github.com/PrairieLearn/PrairieLearn/blob/6ca04597e0e28f0b2c1acb1ed4dc49880b128132/apps/prairielearn/assets/scripts/tsconfig.json#L7

Reply:
Added `bootstrap-table` in [306ccbe](https://github.com/PrairieLearn/PrairieLearn/pull/9725/commits/306ccbe9c0ebd4481747d7e7ddce945db6fff328)3